cetmix_tower_server/views/cx_tower_server_template_view.xml

```diff
@@ -56,7 +57,10 @@
                             <div class="oe_kanban_global_click">
                                 <div class="col-12 pt8 o_kanban_primary_right">
                                     <div>
-                                        <field name="reference" />
+                                        <strong>Servers:</strong>
+                                        <span class="badge badge-primary">
```

Comment:
No need to use badge here, because it's less readable and the entire design of this section is different.